src/lib.rs

```diff
@@ -256,11 +256,7 @@ impl Client {
     /// make sure to take ownership properly of the file descriptors passed
```

Comment:
I can't target in on GitHub, but the safety comment above this still states that it takes ownership, which is now incorrect.

Reply:
Thank you!
I've fixed the doc.